js/indicators/tema.src.js

```diff
@@ -50,6 +49,21 @@ H.seriesType('tema', 'ema',
                 SMA
             );
         },
+        getPoint: function (
+          xVal,
+          tripledPeriod,
+          EMA,
+          EMAlevel2,
+          EMAlevel3,
+          prevEMAlevel3,
```

Comment:
Consider to pack EMA / EMAlevel2/3 / prevEMA into one object to not have so many arguments.